### PR TITLE
Template production env files

### PR DESCRIPTION
**Pull request by willfurnass**
These env files are to help better document what needs setting/changing for production deployments of this site.

@annakrystalli: Which README file should I add docs to?  [README.rst](README.rst) or [README.md](README.md)?

### CODE REVIEW

File: .envs/.production/.postgres.tmpl [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# PostgreSQL
+# ------------------------------------------------------------------------------
+POSTGRES_HOST=postgres
+POSTGRES_PORT=5432
+POSTGRES_DB=reprohack_hub
+# Postgres user should be an alpanumeric string
+POSTGRES_USER=$TMPL_POSTGRES_USER
+# Postgres password should be an alpanumeric string
+POSTGRES_PASSWORD=jvWaPUIpOt6NezC5BBzBQNRWtKL73fNkYxOunmMTdlbUT2lNil4pIWWHEGOrhG4p
```
Comment (davidwilby) [inline .envs/.production/.postgres.tmpl:9]: I assume this is not a used password?

**Comment by annakrystalli** [comment]
At some point I think they need to be merged but for now I reckon `README.md` is best

**Comment by annakrystalli** [comment]
Let me know when you think you are ready to merge

File: .envs/.production/.postgres.tmpl (hunk 1 above)
Comment (willfurnass) [reply .envs/.production/.postgres.tmpl]: Doh!  Fixed in an amended, force-pushed commit to this branch.  I've also changed that password on the dev VM and in an Ansible Vault.

**Comment by annakrystalli** [comment]
Is this ready to merge @willfurnass ? Dying to see if it will resolve #70 😃

**Comment by annakrystalli** [comment]
Let me know when #79 is ready to merge. Atm it's showing that it's failing?

**Pull request closed without merging**